tests/queries/0_stateless/03221_script_line_number.sql

```diff
@@ -0,0 +1,34 @@
+-- Tags: no-parallel
```

Comment:
Please add a comment why the test must not run in parallel (otherwise, future programmers will need to find it out the hard way)

EDIT: Okay, it is because the test crates three schemas. You could get rid of `no-parallel` by not creating schemas at all. Instead create three tables in the default schema (each test run gets its own randomly generated database schema)

Reply:
Could this test file please also check the new setting?